nixos/modules/services/web-apps/suwayomi-server.nix

```
@@ -72,7 +72,7 @@ in
               };
 
               basicAuthEnabled = mkEnableOption ''
-                Add basic access authentication to Suwayomi-Server.
+                adding basic access authentication to Suwayomi-Server.
```

```suggestion
                basic access authentication to Suwayomi-Server.
```
``Wether to enable adding`` sounds strange to me